src/components/checkbox/checkbox.template.tsx

```diff
@@ -12,17 +14,20 @@ export const CheckboxTemplate: FunctionalComponent<CheckboxTemplateProps> = prop
     return (
         <div class="mdc-form-field ">
             <div
-                class={`
-                        mdc-checkbox
-                        ${props.disabled ? 'mdc-checkbox--disabled' : ''}
-                    `}
+                class={{
+                    'mdc-checkbox': true,
+                    'mdc-checkbox--invalid': props.invalid,
+                    'mdc-checkbox--disabled': props.disabled,
+                    'mdc-checkbox--required': props.required,
+                }}
             >
                 <input
                     type="checkbox"
                     class="mdc-checkbox__native-control"
                     id={props.id}
                     checked={props.checked}
                     disabled={props.disabled}
+                    required={props.required}
```

Comment:
Something makes the checkbox in the existing example required… 🤔

Reply:
Ah… hehe… the checkbox in the example was explicitly set to required already 😂

Reply:
Ah again… Now I see you set it in this PR. I missed that at first, for some reason. Sorry for spamming you with notifications 😄